.gitignore

```diff
@@ -15,3 +15,6 @@ node_modules
 /dist
 
 /nova.yaml
+
+/result
+/.direnv
```

Comment:
Please don't add things like this to a project-level gitignore, as this will lead to a long-term proliferation of bespoke entries in the file.

Instead, we recommend using a [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) in your development environment for things like this.